lib/backtrace/prettifysegfault.cpp

```diff
@@ -1,3 +1,4 @@
+#ifndef __GNUC__
```

Comment:
__GNUC__ is unfortunately defined by clang as well (doesn't really makes sense, but there are reasons etc). To keep this running on clang on OS X, iOS and Windows use "#if !defined(__GNUC__) || defined(__APPLE__)" instead.